commercetools.Sdk/commercetools.Sdk.HttpApi.IntegrationTests/ProductProjections/ProductProjectionsIntegrationTests.cs

```diff
@@ -26,7 +27,7 @@ public void GetProductProjectionById()
             //Arrange
             IClient commerceToolsClient = this.productProjectionsFixture.GetService<IClient>();
             var stagedProduct = this.productProjectionsFixture.productFixture.CreateProduct(true);
-            var publishedProduct = this.productProjectionsFixture.productFixture.CreateProductAndPublishIt(true);
+            var publishedProduct = this.productProjectionsFixture.productFixture.CreateProduct(true, true);
```

Comment:
could you use named parameters here? like

`CreateProduct(withVariants: true, published: true)`

this should make the code more readable what flags are set and also for the other CreateProduct calls please

Reply:
Done